app/views/users/show.html.erb

```diff
@@ -10,4 +10,13 @@
   <section>
     <h1>Bookmarked Segments</h1>
   </section>
+
+  <% if token %>
+    <section>
+      <h1>GitHub Repositories</h1>
+      <% facade.repositories.each do |repo| %>
+        <%= link_to repo.name, repo.html_url %>
+      <% end %>
+    </section>
+  <% end %>
```

Comment:
Again this works, wondering if we can just set an attr_reader for @token the facade.